### ISSUE TITLE
repetition result on cityscapes dataset

### ISSUE THREAD

**Issue by isolution911** [ShuLiu1993/PANet#36]
Has anybody trained PANet on Cityscapes dataset？when I try to train PANet on Cityscapes dataset with hyper-parameters listed in the paper, I only get a 30.26 mask AP on Cityscapes validation set,which is much less than 36.5 mask AP in the paper. Maybe i got something wrong in the configuration file, Would anyone like to share his configuration file on cityscapes?thanks a lot.

**Comment by skingorz** [comment]
Have you solved the problem? I ran into the same problem.
